cmd/flowrunner/testdata/flows/subflow_loop_test.json

```diff
@@ -19,10 +19,10 @@
         {
           "action_uuid": "300f02ba-e0b5-4991-bed6-4c240cdb8743",
           "event": {
-            "contact_uuid": "ba96bf7f-bc2a-4873-a7c7-254d1927c4e3",
             "created_on": "2000-01-01T00:00:00.000000000-00:00",
             "flow_uuid": "a8d27b94-d3d0-4a96-8074-0f162f342195",
-            "type": "flow_entered"
+            "parent_run_uuid": "",
```

Comment:
@nicpottier thoughts on this? I like having only one way of detecting loops... but I don't like breaking our unit tests.